spec/rubocop/cop/rspec/spec_file_path_format_spec.rb

```diff
@@ -281,4 +281,272 @@ class Foo
       RUBY
     end
   end
+
+  shared_context 'with successful require stubbing' do
+    before do
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require).and_call_original
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require)
+        .with('active_support/inflector').and_return(true)
+      allow(TOPLEVEL_BINDING.receiver).to receive(:require)
+        .with(cop_config['InflectorPath']).and_return(true)
+    end
+  end
+
+  shared_context 'with ActiveSupport available' do
+    before do
+      allow(described_class::ActiveSupportInflector).to receive(:available?)
+        .and_return(true)
+    end
+  end
+
+  shared_context 'when using ActiveSupport integration' do
```

Comment:
It looks like this context is shared between (almost) all the specs you add. Should we try and add e.g. a `describe ActiveSupport integration` block around it all? I know that that would violate `RSpec/NestedGroups`, but let’s just disable that for now.

Reply:
Yes, that's what I started with when refactoring to use config, I still like it a lot more, I will turn it back into a NestedGroup!

Reply:
Even with 3 lines for the enable/disable and an explanatory comment, this still saves us 9 lines!

Reply:
I was able to cleanly remove all shared contexts